WORKSPACE

```diff
@@ -88,8 +88,7 @@ go_repository(
     name = "com_github_golangci_golangci-lint",
     build_file_generation = "on",
     importpath = "github.com/golangci/golangci-lint",
-    strip_prefix = "golangci-lint-4570c043a9b56ccf0372cb6ba7a8b645d92b3357",
-    urls = ["https://github.com/golangci/golangci-lint/archive/4570c043a9b56ccf0372cb6ba7a8b645d92b3357.zip"],
+    tag = "v1.13",
```

Comment:
Why `v1.13` here? Any reason we shouldn't be using the latest `v1.15.0` tag?

Reply:
No reason apart from the fact that this version is also used in cluster-api repo: https://github.com/kubernetes-sigs/cluster-api/blob/e9f80414db98ed2bf35f23100f0da15a601a6fc3/WORKSPACE#L38